Rework Lightspeed docs (#4524)
theforeman/foreman-documentation: guides/common/modules/con_monitoring-hosts-by-using-insights-in-cloud.adoc

For more information, see xref:deploying-{insights-id}-client-on-registered-hosts_{context}[].
ifdef::satellite[]
* If you register your host to a new {ProjectServer}, reregister the host to {Insights} to avoid creating duplicate entries.
For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Lightspeed].
endif::[]
* You can change the default schedule for running `insights-client` by configuring `insights-client.timer` on a host.
ifdef::satellite[]
For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Lightspeed_.
endif::[]

.Troubleshooting
* You can find the logs for all plugins in the `/var/log/foreman/production.log` file.
* If you have problems connecting to {Insights}, ensure that your certificates are up-to-date.

> Refresh your subscription manifest to update your certificates.

Reviewer: I don't see a reason to highlight here that production.log contains logs "for all plugins." (It does, but it's not really relevant.)

Outdated certificates are only one thing that can happen. For problems connecting hosts to Insights, may also want to mention
- make sure the host_registration_insights parameter is set to true for the host (link to that part, ofc)
- make sure the host has access to RHEL BaseOS and AppStream repositories so that insights-client is available to install

:_mod-docs-content-type: CONCEPT

[id="monitoring-hosts-by-using-{insights-id}-in-cloud"]
= Monitoring hosts by using {Insights} in {RHCloud}

[role="_abstract"]
You can use the {Insights} advisor service in the {RHCloud} to assess system health on hosts running {RHEL}.

> You can use the {Insights} dashboard in the {RHCloud} to quickly analyze recommendations for remediation, vulnerabilities, and more.

Reviewer 2: ```suggestion
You can use the {Insights} dashboard in the {RHCloud} to quickly analyze recommendations for remediation and vulnerabilities.
```

Just a suggestion. Feels a bit marketing-y. I am OK either way.

If you use {Insights} in the {RHCloud}:

* You can use {Insights} services online in the {RHCloud}.
* The Insights client collects data of your hosts and {Project} forwards the data through the Insights client to the {RHCloud} to calculate recommendations.
* You can view and remediate those recommendations both in the {ProjectWebUI} and in the {RHCloud}.
* {Project} uploads host inventory reports to the {RHCloud}.
* You have control over how much information is included in those reports.
* You cannot use local {insights-iop}.

.Prerequisites

> * For new {Project} hosts, install and configure the Insights client during host registration to {Project}.

Reviewer 2: I do not like "{Project} hosts". It's unclear if that is any Smart Proxy Server or any host. Reading on, it's clear that it's both technically. I suggest to reword it:

```suggestion
* Install and configure the Insights client during host registration to {Project} in your registration and provisioning workflows.
```

AFAIK each supported provisioning workflow registers the host to Foreman+Katello too. The alternative is to split this into three parts:

* Registering existing hosts (this line+21)
* Provisioning hosts (kinda missing/not that explicit)
* Enabling on hosts via Ansible (see line 22)

For more information, see xref:deploying-insights-client-on-registered-hosts_{context}[].
ifdef::satellite[]
* If you register your host to a new {ProjectServer}, reregister the host to {Insights} to avoid creating duplicate entries.
For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red{nbsp}Hat Lightspeed].
endif::[]
* You can change the schedule for running the Insights client by configuring `insights-client.timer` on a host.
ifdef::satellite[]
For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red{nbsp}Hat Lightspeed_.
endif::[]

.Troubleshooting
* You can find the logs in the `/var/log/foreman/production.log` file.
* If you have problems connecting to {Insights}, ensure that your certificates are up-to-date.

> Refresh your subscription manifest to update your certificates.

Reviewer 2: I do not understand which certificates this refers to: Red Hat manifest on Foreman+Katello (aka https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Updating_and_Refreshing_Red_Hat_Subscription_Manifests_content-management)? certificates on clients (aka `subscription-manager refresh`)?

@jeremylenz Could you please weigh in?

Reviewer: Refreshing a manifest updates the upstream consumer identity certificate in the Candlepin owner, which is used by foreman_rh_cloud as credentials to communicate with console dot.
